tools/send-clean.sh

```diff
@@ -67,6 +67,16 @@ fi
 if [ "${BASENAME##*.}" = "gz" ]
 then
   ${GZIP_BINARY} --force --quiet --decompress ${FILENAME}
+  UNZIP_COMMAND_RET=$?
+  if [ $UNZIP_COMMAND_RET -ne 0 ]
+  then
+    echo "ERROR: failed to decompress inventory in ${FILENAME} az gzip, putting it in the failed directory"
+    mv "${FILENAME}" "${FAILEDDIR}/`basename ${FILENAME}`-$(date --rfc-3339=date)"
+    mv "${FILENAME_SIGN}" "${FAILEDDIR}/`basename ${FILENAME_SIGN}`-$(date --rfc-3339=date)"
```

Comment:
There is not always a signature file, you should avoid errors here with a [ -f ] &&